devices/debian_wifi.py

```diff
@@ -1,88 +1,109 @@
-import re
+import re, time
 import debian
 import pexpect
+from countrycode import countrycode
+from lib.wifi import wifi_client_stub
 
-class DebianWifi(debian.DebianBox):
+class DebianWifi(debian.DebianBox, wifi_client_stub):
     '''Extension of Debian class with wifi functions'''
 
     model = ('debianwifi')
+    iface_wlan = "wlan0"
+    iface_wlan1 = "wlan1"
 
-    iface_wlan = "wlan1"
+    def disable_and_enable_wifi(self, iface):
+       self.set_link_state(iface,"down")
+       self.set_link_state(iface,"up")
```

Comment:
Can you fix the space after the , ?

I would just all disable_wifi() and enable_wifi() incase those functions ever do something else.